### PR TITLE
SUS-4080 | remove handling of video_info.premium column in app code

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-4080

### CODE REVIEW

File: extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php [hunk 1]
```diff
@@ -415,6 +378,8 @@ protected static function getMemcKey( $videoTitle ) {
 	 * save to cache
 	 */
 	protected function saveToCache() {
+		$cache = [];
```
Comment (pwojcik86) [inline extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php:381]: Is this related to premium videos or just a fix?

**Review by pwojcik86 — APPROVED** [review]

File: includes/wikia/services/MediaQueryService.class.php [hunk 2]
```diff
@@ -398,7 +398,6 @@ public static function getRecentlyUploadedAsMediaTable( $limit ) {
 	 *
 	 * @param string $type What type of videos to return.  Valid options are:
 	 *                     - all     : Show all videos (DEFAULT)
-	 *                     - premium : Show only premium videos
```
Comment (mszabo-wikia) [inline includes/wikia/services/MediaQueryService.class.php]: let's remove `type` param if it only handles 1 type

**Review by mszabo-wikia — CHANGES_REQUESTED** [review]

File: extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php (hunk 1 above)
Comment (macbre) [reply extensions/wikia/VideoHandlers/videoInfo/VideoInfo.class.php:381]: Just a fix, the variable was not initialised properly.

File: includes/wikia/services/MediaQueryService.class.php (hunk 2 above)
Comment (macbre) [reply includes/wikia/services/MediaQueryService.class.php]: Fixed in 4c824c7

**Review by mszabo-wikia — APPROVED** [review]
